packages/design-system-react/src/components/text/Text.tsx

```diff
@@ -15,6 +15,7 @@ export const Text: React.FC<TextProps> = ({
   ellipsis,
   as,
   color = TextColor.TextDefault,
+  style,
```

Comment:
Move this to it's own PR. Also create an issue to allow spreading of props to the `Text` component